src/EnergyPlus/Autosizing/SystemAirFlowSizing.cc

```diff
@@ -777,24 +777,63 @@ Real64 SystemAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, bo
                     }
 
                 } else {
-                    auto &sysSizPeakDDNum = state.dataSize->SysSizPeakDDNum(this->curSysNum);
+                    if (this->curOASysNum > 0) {
+                        if (this->oaSysEqSizing(this->curOASysNum).AirFlow) {
+                            // Parent object sets system flow rate
+                            this->autoSizedValue = this->oaSysEqSizing(this->curOASysNum).AirVolFlow;
+                        } else if (this->oaSysEqSizing(this->curOASysNum).CoolingAirFlow || this->oaSysEqSizing(this->curOASysNum).HeatingAirFlow) {
+                            // Parent object sets heating flow rate
+                            this->autoSizedValue = std::max(this->oaSysEqSizing(this->curOASysNum).CoolingAirVolFlow,
+                                                            this->oaSysEqSizing(this->curOASysNum).HeatingAirVolFlow);
+                        } else if (outsideAirSys(this->curOASysNum).AirLoopDOASNum > -1) {
+                            this->autoSizedValue =
+                                this->airloopDOAS[outsideAirSys(this->curOASysNum).AirLoopDOASNum].SizingMassFlow / state.dataEnvrn->StdRhoAir;
+                        } else {
+                            this->autoSizedValue = this->finalSysSizing(this->curSysNum).DesOutAirVolFlow;
+                        }
+                    } else {
```

Comment:
The code change is from here to line 180 above. The below is just git confusion.